xml/System.Web.Script.Serialization/JavaScriptSerializer.xml

```diff
@@ -16,7 +16,7 @@
   <Interfaces />
   <Docs>
     <summary>
-      <see href="https://www.newtonsoft.com/json">Json.NET</see> should be used for serialization and deserialization. Provides serialization and deserialization functionality for AJAX-enabled applications.</summary>
+      The APIs in the <see cref="N:System.Text.Json" /> namespace should be used for serialization and deserialization. Provides serialization and deserialization functionality for AJAX-enabled applications.</summary>
```

Comment:
Isn't this statement true ONLY for .NET Framework 4.6.1 or later?

Reply:
Wherever we were recommending Json.NET we should  be recommending STJ, so I only changed that. That said, the structure of this section doesn't really make sense. If I'm reading it correctly, the "Provides..." sentence applies to this class, and so the first sentence was probably added later to say "Use Json.NET instead of this class" but the way it was added doesn't make that intention clear.

Reply:
> Use Json.NET instead of this class

Yes, I recall switching from `JavaScriptSerializer` to JSON.NET years ago for performance reasons. At one point, there was messaging from Microsoft that completely discouraged the use of `JavaScriptSerializer`.

Reply:
@ahsonkhan In this doc should we say use Newtonsoft.json for earlier than 4.6.1 and System.Text.Json for 4.6.1 and later?

Reply:
@layomia @Jozkee @steveharter can you please help answer this question?

Reply:
@steveharter @layomia @Jozkee ping. What is the correct sentence to use here?

Reply:
Yes, `System.Text.Json` targets `netstandard2.0` which makes it compatible only with .NET Framework 4.6.1 or later. 

> should we say use Newtonsoft.json for earlier than 4.6.1 and System.Text.Json for 4.6.1 and later

That seems to be the most accurate. Why is that advice on the summary and not as a warning in the `JavaScriptSerializer` page?

Reply:
> Why is that advice on the summary and not as a warning in the JavaScriptSerializer page?

No way to know why it's there or who put it there -- it's already there in the earliest commit in the repo.

I agree it's out of place where it is, I'll move it to Remarks.

Reply:
> I'll move it to Remarks.

If we only have it in remarks, it won't show up in IntelliSense for folks using IDEs like VS. We may want to keep the note in the summary as well as the remarks.